### DISCUSSION TITLE
Conditional rules for GraphQL query

### DISCUSSION THREAD

**Discussion by Stalex89** [solspace/craft-freeform#727] [Feature Requests]
Hello, 

Could you please clarify how to fetch field conditional rules with GraphQL (if it's even possible)?

I applied Conditional rule on field to appear after some value is selected on previous field, but cannot find a way to query it with GraphQL. Also cannot find GraphQL example of it in documentation [here](https://docs.solspace.com/craft/freeform/v4/overview/conditional-rules/) or [here](https://docs.solspace.com/craft/freeform/v4/headless/nextjs/).

Thank you in advance!

**Comment by kjmartens** [comment]
Hi @Stalex89,

Unfortunately this currently isn't available, but it's something we plan to implement in the near future. 🙂

**Reply by Stalex89** [reply]
Thank you for the quick reply, looking forward for this crucial feature in the future updates 🙂

**Comment by kjmartens** [comment]
This has been added now in Freeform 4.1.5+ 🙂

**Reply by Stalex89** [reply]
@kjmartens Thank you very much! 😄
